cache.go

```diff
@@ -24,6 +24,7 @@ type Cache interface {
 	fmt.Stringer
 	Set(key string, value interface{}, ttl time.Duration)
 	Get(key string) (interface{}, bool)
+	GetExpiration(key string) time.Time
```

Comment:
Do we have a reason to avoid returning (exp time, ok bool) similar to Get instead of zero-value time if it is not found? I can think of potential errors users can make by trying to retrieve and store keys in one fragment of code and iterating over it in another with time.Before check, which likely led to an error.

Reply:
It is fixed in #10.